src/edgepi/calibration/edgepi_eeprom.py

```diff
@@ -224,9 +226,10 @@ def __generate_list_of_pages(self, mem_addr: int = None, data: list = None):
             page_n = [data_remainder[byte:byte+EEPROMInfo.PAGE_SIZE.value] \
                       for byte in range(0,len(data_remainder), EEPROMInfo.PAGE_SIZE.value)]
             page_n.insert(0, page_1)
+            self.log.debug(f"__generate_list_of_pages: {len(page_n)} pages generated")
         return page_n
 
-    def read_memory(self, start_addrx: int = None, length: int = None):
+    def read_memory(self, start_addrx: int = 0, length: int = None):
```

Comment:
why is length's default `None` ?

Reply:
I want to throw an error when the user forgets to pass the length